src/components/date-input/index.md.njk

```diff
@@ -85,7 +85,7 @@ Say ‘Enter [whatever it is]’. For example, ‘Enter your date of birth’.
 
 #### If the date is incomplete
 
-Highlight the day, month or year field where the information is missing. If more than one field is missing information, highlight the date input as a whole.<br>
+Highlight the day, month or year field where the information is missing. If more than one field is missing information, highlight the fields the user needs to fill in.<br>
```

Comment:
@calvin-lau-sig7 Thanks for reviewing! Should we add content about the field highlighting matching the error message? Or just leave this comment open in case we iterate later?